mautic/mautic
# Add search by company fields for list view/API

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | https://github.com/mautic/developer-documentation/pull/145
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR bring search by fields for companies (UX/API), same like contacts do.


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to companies,  create companies with some data - name, city etc
3. Try search from field: city:"Your City" etc
4. You should see suggestion for that fields in the search field too
5. Same search feature you can use in API

```
$companyApi       = $api->newApi('companies', $auth, $apiUrl);
$response = $companyApi->getList('companyaddress1:test');
die(print_r($response));
```
